packages/gatsby-theme-docs/gatsby-ssr.js

```diff
@@ -20,6 +20,9 @@ const iconLightDigest = createContentDigest(
   fs.readFileSync(require.resolve('./static/favicon-light-32x32.png'))
 );
 
+const patchedLobotomizedOwlSelector = '> *:not(style) ~ *:not(style)';
+const lobotomizedOwlSelectorRegex = />\s?\*\s?\+\s?\*/g;
```

Comment:
since we're here to make this more robust the regex could also allow more than one whitespace-ish character. 
```suggestion
const lobotomizedOwlSelectorRegex = />\s*\*\s*\+\s*\*/g;
```